tools/cli/README.md

```diff
@@ -150,3 +150,18 @@ Use option `--workflowidreusepolicy` or `--wrp` to configure the workflow id re-
 ```
 Terminating a running workflow execution will record a WorkflowExecutionTerminated event as the closing event in the history. No more decision tasks will be scheduled for a terminated workflow execution.  
 Canceling a running workflow execution will record a WorkflowExecutionCancelRequested event in the history, and a new decision task will be scheduled. The workflow has a chance to do some clean up work after cancellation.
+
+- Restart, reset workflow
```

Comment:
You need to put a bunch of spaces at the end of this line so that it puts a newline before the next paragraph

Reply:
Got it